doc/src/manual/getting-started.md

```diff
@@ -118,6 +120,9 @@ julia [switches] -- [programfile] [args...]
  --color={yes|no}          Enable or disable color text
  --history-file={yes|no}   Load or save history
 
+ --depwarn={yes|no|error}  Enable or disable syntax and method deprecation warnings ("error" turns warnings into errors)
+ --warn-overwrite={yes|no} Enable or disable method overwrite warnings
+
  --compile={yes|no|all|min}Enable or disable JIT compiler, or request exhaustive compilation
```

Comment:
appears that we should add a couple spaces, or split this one over multiple lines?